tests/test_loader.py

```diff
@@ -225,11 +201,14 @@ def test():
     """
 
     with pytest.raises(ImportError):
-        import inmanta_bad_unit_test  # NOQA
+        import inmanta_plugins.inmanta_bad_unit_test  # NOQA
 
     cl.deploy_version([get_module_source("inmanta_plugins.inmanta_bad_unit_test", code)])
 
-    assert "ModuleNotFoundError: No module named 'badimmport'" in caplog.text
+    with pytest.raises(ModuleNotFoundError):
+        import inmanta_plugins.inmanta_bad_unit_test  # NOQA
```

Comment:
This was previously relying on the fact that the reloading would try importing, we have to explicitly call `import` now.